cheatsheets/gleam-for-go-users.md

```diff
@@ -0,0 +1,1452 @@
+---
+layout: page
+title: Gleam for Go users
+subtitle: Hello Gophers!
+---
+
+- [Comments](#comments)
+- [Variables](#variables)
+  - [Match operator](#match-operator)
+  - [Variables type annotations](#variables-type-annotations)
+- [Functions](#functions)
+  - [Exporting functions](#exporting-functions)
+  - [Function type annotations](#function-type-annotations)
+  - [Referencing functions](#referencing-functions)
+  - [Labelled arguments](#labelled-arguments)
+- [Operators](#operators)
+- [Constants](#constants)
+- [Blocks](#blocks)
+- [Data types](#data-types)
+  - [Strings](#strings)
+  - [Tuples](#tuples)
+  - [Lists](#lists)
+  - [Maps](#maps)
+  - [Numbers](#numbers)
+- [Flow control](#flow-control)
+  - [Case](#case)
+  - [Piping](#piping)
+  - [Error Handling](#error-handling)
+- [Type aliases](#type-aliases)
+- [Custom types](#custom-types)
+  - [Records](#records)
+  - [Unions](#unions) IN PROGRESS
+  - [Opaque custom types](#opaque-custom-types) IN PROGRESS
+- [Modules](#modules) IN PROGRESS
+  - [Imports](#imports) IN PROGRESS
+  - [Named imports](#named-imports) IN PROGRESS
+  - [Unqualified imports](#unqualified-imports) IN PROGRESS
+- [Architecture](#architecture) IN PROGRESS
+
+## Comments
+
+### Go
+
+In Go, comments are written with a `//` prefix.
+
+```Go
+// Hello, Joe!
+```
+
+Multi line comments may be written like so:
+
+```Go
+/*
+ * Hello, Joe!
+ */
+```
+
+In Go, top level declarations can be annotated with a comment directly above it to create documentation for that symbol.
+
+```Go
+// This is a special interface
+type Foo interface {}
```

Comment:
Could you swap "foobar" for something that isn't a horrors-of-war reference please 🙏